### PR TITLE
Memory Game

### LINKED ISSUE #28: Project : Memory game
Basically It is a card Game where you will be tested your memory.

### PR DESCRIPTION
**Pull request by 2devyank**
## Description

- Its a memory game which will test your memory

Closes #28 

## Add GIF/ScreenShot 
Untitled design (1)

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [x] You have created a new branch for contribution.
- [x] Added README.md for your project in a specified folder.
- [x] Add Demo in GIF or Screenshot form.
- [x] You are Registered Officially for HacktoberFest 2021 as Partcipant.
- [x] Followed Contribution Guidelines.

## Issue 
 your issue no. #28

### CODE REVIEW

**Comment by 2devyank** [comment]
@soumik2012 @Harshil-Jani I think now everything is fine please check. Sorry for the inconvenience.

**Review by Harshil-Jani — APPROVED** [review]
We are good to go :rocket:

File: MemoryGame/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Untitled design](https://user-images.githubusercontent.com/78840243/135575003-f9773cfe-51e5-4300-a416-a72393f49084.gif)
+![Untitled design (1)](https://user-images.githubusercontent.com/78840243/135575411-3171de1d-4b36-4d80-97d6-8f4ca9e5aab7.gif)
```
Comment (soumik2012) [inline MemoryGame/README.md:1]: Write your author-name in readme.md

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Code is fine to add your author name in the readme.md file

**Comment by 2devyank** [comment]
@soumik2012 I have added please check

**Review by soumik2012 — APPROVED** [review]
LGTM 💯

**Pull request merged**